cromper/tests/test_compilation.py

```diff
@@ -4,7 +4,7 @@
 
 import unittest
 
-from cromper.compiler_wrapper import CompilationResult
+from cromper.cromper.wrappers.compiler_wrapper import CompilationResult
```

Comment:
can't these be just "cromper.wrappers" and so on? the double cromper is kinda ugly

Reply:
Urgh this was vscode automatically adjusting the import when I moved the files, will change